walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -19,9 +19,22 @@
  */
 public class Greeter {
   /**
-   * Returns a greeting for the given name.
+   * Returns a greeting for the given name. 
+   * @param name The name of the user
+   * @return Hello <name>, will trim whitespace and non letters, 
+   * will just return hello is name empty or null
    */
   public String greet(String name) {
```

Comment:
you can make this method static if it turns out that you don't have any state in this class. This way you don't have to create an instance of the class every time you need to use the method